ui/src/containers/App.js

```diff
@@ -28,98 +21,22 @@ addLocaleData([...locale_en, ...locale_fr]);
 
 class App extends Component {
   render() {
-    const intl = messages[this.props.language];
-    const applications = [];
-
-    const help = [
-      {
-        label: intl.about,
-        onClick: () => {
-          this.props.history.push('/about');
-        }
-      }
-    ];
-
-    const user = {
-      name: this.props.user && this.props.user.username,
-      actions: [{ label: intl.log_out, onClick: this.props.logout }]
-    };
-
-    const sidebar = {
-      expanded: this.props.sidebar.expanded,
-      actions: [
-        {
-          label: intl.nodes,
-          icon: <i className="fas fa-server" />,
-          onClick: () => {
-            this.props.history.push('/nodes');
-          },
-          active:
-            matchPath(this.props.history.location.pathname, {
-              path: '/',
-              exact: true,
-              strict: true
-            }) ||
-            matchPath(this.props.history.location.pathname, {
-              path: '/nodes',
-              exact: true,
-              strict: true
-            })
-        }
-      ]
-    };
-
-    const navbar = {
-      onToggleClick: this.props.toggleSidebar,
-      toggleVisible: true,
-      productName: intl.product_name,
-      applications,
-      help,
-      user: this.props.user && user
-    };
-
-    const theme = {
-      brand: {
-        primary: '#283593'
-      }
-    };
-
     return (
       <IntlProvider
         locale={this.props.language}
         messages={messages[this.props.language]}
       >
         <Switch>
           <Route path="/login" component={Login} />
-          <ThemeProvider theme={theme}>
-            <Layout sidebar={sidebar} navbar={navbar}>
-              <PrivateRoute exact path="/nodes" component={NodeList} />
-              <PrivateRoute exact path="/about" component={Welcome} />
-              <PrivateRoute exact path="/" component={NodeList} />
-            </Layout>
-          </ThemeProvider>
+          <Layout />
```

Comment:
Wouldn't it be cleaner/safer to do:

```suggestion
          <Route component={Layout} />
```

And include a `Switch` in `Layout`? I think it would best contribute to the separation of concerns and decoupling between `App` and `Layout`.